.github/workflows/defaultLabels.yml

```diff
@@ -8,7 +8,7 @@ jobs:
    label-issues:
       runs-on: ubuntu-latest
       steps:
-         - uses: actions/stale@v9
+         - uses: actions/stale@v10
```

Comment:
Side note and we should add this as separate pr but let’s SHA pin this one as well.